### ISSUE TITLE
Second Texture Coord Set doesn't seem to be used

### ISSUE THREAD

**Issue by stevk** [bghgary/glTF-Asset-Generator#119]
The second texture coord set is set, but doesn't appear on the model. There is probably some missing part to how it is declared.

**Comment by stevk** [comment]
From what I'm seeing, only one index can be set per BaseColorTexture, only one BaseColorTexture can be set per material, and only one material can be set per primitive. The closest I could find to using two UV sets was making multiple primitives and setting a different material on each.

Is this expected? Or should it be possible to set multiple textures on one primitive?

**Comment by kcoley** [comment]
Yeah, this is expected behavior.  The best you can do is assign the different texture coordinates to the metallic roughness, normal, occlusion, etc. textures which will render them together.

**Comment by stevk** [comment]
I've implemented an emissive texture and I think I've found an issue with how the UV index is set.

Material.cs Line 148 sets the result to the sampler_index and image_index. but that image_index is used for setting the TexCoord index on line 263. (this is looking at my current branch `stevk/primattributes2`)

**Comment by kcoley** [comment]
I'll take a look

**Comment by kcoley** [comment]
Definitely a bug.  I'll work on fixing this.

**Comment by kcoley** [comment]
PR fix was merged #124
